File: Python/Uri1043_Triangulo.py
numeros = input().split()

a = float(numeros[0])
b = float(numeros[1])
c = float(numeros[2])

if((((a+b) > c) and ((a+c)> b) and ((b+c)> a) and (a > 0) and (b > 0) and (b > 0))):
    perimetro = a + b + c
    print("Perimetro = {:.1f}".format(perimetro))
else:
    areaTrapezio = ((a+b)*c)/2
    print("Area = {:.1f}".format(areaTrapezio))

File: C/Iniciantes/Uri1043_Triangulo.c
#include <stdio.h>

int main(){
	
	float a, b, c;
	float perimetro, areaTrapezio;
	
	scanf("%f %f %f", &a, &b, &c);
	
	if ((((a+b) > c) && ((a+c)> b) && ((b+c)> a) && (a > 0) && (b > 0) && (b > 0))){
		perimetro = a + b + c;
		printf("Perimetro = %.1f\n", perimetro);	
	}
	
	else{
		areaTrapezio = ((a+b)*c)/2;
		printf("Area = %.1f\n", areaTrapezio);
	}	
return 0;
}
